storage/test/multiIO_test.cpp

```diff
@@ -157,9 +168,16 @@ TEST(multiIO_test, no_commit_during_exception)
 
 int main(int argc, char **argv) {
   ::testing::InitGoogleTest(&argc, argv);
+
+#ifdef USE_ROCKSDB
   const string dbPath = "./rocksdb_test";
   dbClient = new Client(dbPath, new KeyComparator(new KeyManipulator()));
   dbClient->init();
```

Comment:
Total nit, but why not put this call outside of the ifdefs, just in case memorydb::Client::init has a non-empty body in the future?